### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Luna-meili**
Added Hacktoberfest contributing guideline link  to the Demos ReadMe, ready for approval? @CaroFG

# Pull Request

## What does this PR do?
Updating the documentation for Hacktoberfest 2022

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue?
- [ ] Have you read the contributing guidelines?
- [ ] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Comment by CaroFG** [comment]
May I suggest putting this section just below the main heading?

**Comment by Luna-meili** [comment]
Thanks very much for the suggestion, it's done : )

File: README.md [hunk 1]
```diff
@@ -21,6 +21,11 @@
 <p align="center">
   <a href="https://slack.meilisearch.com"><img src="https://img.shields.io/badge/slack-Meilisearch-blue.svg?logo=slack" alt="Slack"></a>
 </p>
+<p align="center">
+<h3>🎃 It&rsquo;s Hacktoberfest 2022 @Meilisearch</h3>
+<p><a href="https://hacktoberfest.com">Hacktoberfest</a> is a celebration of the open-source community. This year, and for the third time in a row, Meilisearch is participating in this fantastic event.</p>
+<p>You&rsquo;d like to contribute? Don&rsquo;t hesitate to check out our <a href="https://github.com/meilisearch/meilisearch/blob/main/CONTRIBUTING.md">contributing guidelines</a>.<br /><br />And for the longer, more complete guideline documentation you can follow <a href="https://github.com/meilisearch/.github/blob/main/Hacktoberfest_2022_contributors_guidelines.md">this link</a>.</p>
```
Comment (CaroFG) [inline README.md:27]:
```suggestion
<p>You&rsquo;d like to contribute? Don&rsquo;t hesitate to check out our <a href="https://github.com/meilisearch/meilisearch/blob/main/CONTRIBUTING.md">contributing guidelines</a>.<br /><br />And for the longer, more complete documentation of the Hacktoberfest guidelines you can follow <a href="https://github.com/meilisearch/.github/blob/main/Hacktoberfest_2022_contributors_guidelines.md">this link</a>.</p>
```

[the remark this answers is not in the document]
Comment (Luna-meili) [reply README.md]: Done!

**Review by CaroFG — APPROVED** [review]

**Pull request merged**